examples/native/components/ControlExamples/Array/Array.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+
+interface ArrayProps {
+  list: string[];
+}
+
+export const Array = ({ list }: ArrayProps) => (
+  <View style={styles.container}>
+    {list.map((item) => (
```

Comment:
Could we do something like 

```typescript
    {list.map((item, index) => (
      <Text key={index} style={styles.item}>
        {item}
      </Text>
    ))}
```

here to avoid the warning?

![Screen Shot 2021-07-14 at 21 34 04](https://user-images.githubusercontent.com/6605505/125674563-ae967da9-e526-48b1-858c-35575ab3ebca.png)

Reply:
Yes! I forgot about this, thanks :)